IBAnimatable/PresentExplodeSegue.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  Created by Tom Baranes on 03/04/16.
+//  Copyright © 2016 Jake Lin. All rights reserved.
+//
+
+import UIKit
+
+public class PresentExplodeSegue: UIStoryboardSegue {
+  public override func perform() {
+    destinationViewController.transitioningDelegate = PresenterManager.sharedManager().retrievePresenter(.Explode, interactiveGestureType: .Default)
```

Comment:
This one should not have `interactiveGestureType`, it means it will use the default as `nil`

`destinationViewController.transitioningDelegate = PresenterManager.sharedManager().retrievePresenter(.Explode)`